draft-ietf-httpbis-retrofit.md

```diff
@@ -189,86 +208,91 @@ Date: Sun, 06 Nov 1994 08:49:37 GMT
+The field names in {{date-fields}} have values that can be mapped into Structured Field values by parsing their payload according to {{Section 5.6.7 of HTTP}} and representing the result as a Date.
 
-| Field Name          | Mapped Field Name      |
-|---------------------|------------------------|
-| Date                | SF-Date                |
-| Expires             | SF-Expires             |
-| If-Modified-Since   | SF-If-Modified-Since   |
-| If-Unmodified-Since | SF-If-Unmodified-Since |
-| Last-Modified       | SF-Last-Modified       |
+| Field Name          |
+|---------------------|
+| Date                |
+| Expires             |
+| If-Modified-Since   |
+| If-Unmodified-Since |
+| Last-Modified       |
 {:id="date-fields" title="Date Fields"}
 
-For example, an Expires field could be mapped as:
+For example, an Expires field's value could be mapped as:
 
 ~~~ http-message-new
-SF-Expires: @1659578233
+@1659578233
 ~~~
 
 ## ETags
 
-The field value of the ETag header field can be mapped into the SF-ETag Structured Field by representing the entity-tag as a String, and the weakness flag as a Boolean "w" parameter on it, where true indicates that the entity-tag is weak; if 0 or unset, the entity-tag is strong.
+The field value of the ETag header field can be mapped into a Structured Field value by representing the entity-tag as a String, and the weakness flag as a Boolean "w" parameter on it, where true indicates that the entity-tag is weak; if 0 or unset, the entity-tag is strong.
 
-For example, this:
+For example, this ETag header field:
 
 ~~~ http-message
 ETag: W/"abcdef"
 ~~~
 
-~~~ http-message
-SF-ETag: "abcdef"; w
-~~~
+would have a mapped value of:
 
-If-None-Match's field value can be mapped into the SF-If-None-Match Structured Field, which is a List of the structure described above. When a field value contains "*", it is represented as a Token.
+~~~
+"abcdef"; w
+~~~
 
-Likewise, If-Match's field value can be mapped into the SF-If-Match Structured Field in the same manner.
+If-None-Match's field value can be mapped into a Structured Field value which is a List of the structure described above. When a field value contains "*", it is represented as a Token.
 
+Likewise, If-Match's field value can be mapped into a Structured Field value in the same manner.
 
-For example:
+For example, this If-None-Match field:
 
 ~~~ http-message
-SF-If-None-Match: "abcdef"; w, "ghijkl", *
+If-None-Match: "abcdef"; w, "ghijkl", *
+~~~
+
+would have a mapped value of:
+
+~~~
+"abcdef"; w, "ghijkl", *
 ~~~
 
 
 ## Cookies
 
-The field values of the Cookie and Set-Cookie fields {{COOKIES}} can be mapped into the SF-Cookie Structured Field (a List) and SF-Set-Cookie Structured Field (a List), respectively.
+The field values of the Cookie and Set-Cookie fields {{COOKIES}} can be mapped into a Structured Fields Lists.
```

Comment:
s/a Structured Fields Lists/Structured Fields Lists/?

(Using "Structured Fields" as an adjective also reads super weird to me, but I'll defer to you on what the right adjective form is here. :-) )